src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -288,15 +252,16 @@ protected function setup_messages( array $events ) {
 				unset( $filters['form_submit'] );
 			}
 
-			$filters = array_values( $filters );
```

Comment:
This won't work as expected if you remove the `array_values`, as `empty( [ 'foo' => null, 'bar' => null] ) ===  false`

Also, since we're consolidating, can we move this into the consolidated `use_ff_link `?